Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -228,9 +228,9 @@ final class RewardsCollectionViewController: UICollectionViewController {
       ?|> \.isActive .~ !isHidden
   }
 
-  private func goToPledge(project: Project, reward: Reward, refTag _: RefTag?) {
+  private func goToPledge(project: Project, reward: Reward, refTag: RefTag?) {
     let pledgeViewController = PledgeViewController.instantiate()
-    pledgeViewController.configureWith(project: project, reward: reward)
+    pledgeViewController.configureWith(project: project, reward: reward, refTag: refTag)
```

Comment:
Hmmm, I *think* it's not necessary. The `updateBackingPaymentSource` mutation doesn't currently accept a `refParam`. I'd imagine that for updating your backing, the backend would automatically set the referrer as `update_pledge` or something like that, since there's no real "referrer" for updating a pledge. I'll double check with product though. Regardless I think if we wanted to pass the `refParam` with the `updateBackingPaymentSource` mutation that would be in a separate PR.